### ISSUE TITLE
updater.ex gives -2 and does not update

### ISSUE THREAD

**Issue by axtens** [peberlein/WEE#3]
```
C:>eui
Euphoria Interpreter v4.1.0 development
   64-bit Windows, Using System Memory
   Revision Date: 2015-02-02 14:18:53, Id: 6300:57179171dbed

ERROR: Must specify the file to be interpreted on the command line
```
```
C:>eui updater.ex
=== WEE Source Code Updater ===
This will overwrite any local changes to the source files.
Press a key to continue, or 'q' to quit.
-2
Failed to download manifest.json

Press any key to exit, then try updating again.
```
The manifest ```http://rawgit.com/peberlein/WEE/master/manifest.json``` downloads fine in browser.

**Comment by peberlein** [comment]
Looks like rawgit.com started redirecting http to https: https://rawgit.com/news/https-only  (The reason to use rawgit.com was to work around github also redirecting to https.)  Until euphoria supports https, there's nothing that can be done to fix it.

**Comment by axtens** [comment]
So clone and build?

**Comment by peberlein** [comment]
Yes, that is best way now.  I suppose I could change update.ex to shell out to wget or curl.  I'll think about it.

**Comment by peberlein** [comment]
Actually I changed updater.ex to use libcurl or wininet to download files via https.  Seems to work, but minimally tested.  You'll need to download updater.ex manually once, but after that you should be fine.

Fixed by commit 798a07c
